src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -390,4 +388,44 @@ Error({Kind:ErrorKind.BadLanguageCode})
 "2023-12-21T12:34:56.789Z"
 
 >> Text(DateTimeValue("12:34:56"),"yyyy-mm-dd hh:mm:ss")
-"1899-12-30 12:34:56"
\ No newline at end of file
+"1899-12-30 12:34:56"
+
+>> Text(1234.5678, "[$-fr-FR]# ##0,00", "en-US")
+"1,234.57"
+
+>> Text(1234.5678, "[$-fr-FR]# ##0,00", "vi-VI")
+"1.234,57"
+
+>> Text(1234.5678, "[$-en-US]#,##0.00", "en-US")
+"1,234.57"
+
+>> Text(1234.5678, "[$-pt-BR]#.##0,00", "en-US")
+"1,234.57"
+
+>> Text(1234.5678, "[$-pt-BR]#.##0,00", "fr-FR")
+"1 234,57"
+
+>> Text(1234.5678, "[$-en-US]#,##0.00", "pt-BR")
```

Comment:
`ParseJSON("""\u0020""")` is the same thing as a single space, so there's no need to use the JSON function for that. It would be interesting to use it to create the French thousands separator, though (U+202F).

Reply:
updated here.